simple-git/src/lib/utils/task-options.ts

```diff
@@ -24,6 +24,8 @@ export function appendTaskOptions<T extends Options = Options>(
          commands.push(value);
       } else if (filterPrimitives(value, ['boolean'])) {
          commands.push(key + '=' + value);
+      } else if (Array.isArray(value)) {
+         commands.push(...value.map((v) => key + '=' + v));
```

Comment:
Please can this use a `filterPrimitives(v, ['string', 'number'])`

Reply:
I'm not sure I understood. Why would I do that? Wouldn't that prevent string and number values from being parsed? Or did you mean:

```ts
      if (isPathSpec(value)) {
         commands.push(value);
      } else if (filterPrimitives(value, ['boolean'])) {
         commands.push(key + '=' + value);
      } else if (filterPrimitives(value, ['string', 'number'])) {
         commands.push(...value.map((v) => key + '=' + v));
      } else {
         commands.push(key);
      }
```